pkg/images/images.go

```diff
@@ -1,9 +1,11 @@
 package images
 
 import (
+	"github.com/sirupsen/logrus"
```

Comment:
Can you move this down to where the other 3rd party imports are? This is usually an ide function of goimports, goreturns, etc.